src/content/css-geometry-tools.js

```diff
@@ -61,33 +55,27 @@ function addMargin(elem, rect, win) {
 /**
  * Returns a DOMRect for the content rect of the element, in float CSS pixels.
  * Returns an empty rect if the object has zero or more than one client rects.
- * @param  {object} elem The DOM element whose content rect to compute.
- * @return {object}      The element's content rect.
  */
-export function getContentRect(elem) {
-  const clientRects = elem.getClientRects();
+export function getContentRect(element: HTMLElement): DOMRectInterface {
+  const clientRects = element.getClientRects();
   if (clientRects.length !== 1) {
     return new DOMRect();
   }
 
   const borderRect = clientRects[0];
-  const win = elem.ownerDocument.defaultView;
-  return subtractPadding(elem, subtractBorder(elem, borderRect, win), win);
+  return subtractPadding(element, subtractBorder(element, borderRect));
 }
 
 /**
  * Returns a DOMRect for the margin rect of the element, in float CSS pixels.
  * Returns an empty rect if the object has zero or more than one client rects.
- * @param  {object} elem The DOM element whose margin rect to compute.
- * @return {object}      The element's margin rect.
  */
-export function getMarginRect(elem) {
-  const clientRects = elem.getClientRects();
+export function getMarginRect(element: HTMLElement): DOMRectInterface {
+  const clientRects = element.getClientRects();
   if (clientRects.length !== 1) {
     return new DOMRect();
   }
 
   const borderRect = clientRects[0];
-  const win = elem.ownerDocument.defaultView;
-  return addMargin(elem, borderRect, win);
+  return addMargin(element, borderRect);
```

Comment:
I flow typed the Reorderable component, and this file ended up getting hit as well. These look like they were copy and pasted over from Gecko's codebase where we care about things like which window element something is in. I was having a lot of trouble getting it typed correctly, so I made the decision to simplify the code and strip out the windows.